code/game/machinery/smelter.dm

```diff
@@ -123,6 +123,8 @@
 				var/obj/item/stack/material/S = smelting
 				total_material *= S.get_amount()
 
+			total_material *= scrap_multiplier	//	Syzygy edit - Nerfs recycling
```

Comment:
Doing this and re-enabling lathe cycling at a measly 5% less efficiency makes the smelter completely worthless.